docker/mysql/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mysql:8.0.17
+ENV MYSQL_ROOT_PASSWORD root
```

Comment:
ENV structure can be merged as one command only. Please check this topic: https://stackoverflow.com/questions/45529121/dockerfile-setting-multiple-environment-variables-in-single-line